composer.json

```diff
@@ -11,51 +11,29 @@
     ],
     "homepage": "http://www.yiiframework.com/",
     "license": "BSD-3-Clause",
-    "authors": [
-        {
-            "name": "Qiang Xue",
-            "email": "qiang.xue@gmail.com",
-            "homepage": "http://www.yiiframework.com/",
-            "role": "Founder and former project lead"
-        },
-        {
-            "name": "Alexander Makarov",
-            "email": "sam@rmcreative.ru",
-            "homepage": "http://rmcreative.ru/",
-            "role": "Core framework development"
-        }
-    ],
     "support": {
-        "source": "https://github.com/yiisoft/cache-memcached",
-        "issues": "https://github.com/yiisoft/cache-memcached/issues",
+        "issues": "https://github.com/yiisoft/cache-memcached/issues?state=open",
         "forum": "http://www.yiiframework.com/forum/",
         "wiki": "http://www.yiiframework.com/wiki/",
-        "irc": "irc://irc.freenode.net/yii"
+        "irc": "irc://irc.freenode.net/yii",
+        "source": "https://github.com/yiisoft/cache-memcached"
     },
-    "minimum-stability": "dev",
-    "prefer-stable": true,
     "require": {
-        "yiisoft/cache": "^3.0@dev",
-        "ext-memcached": "*"
+        "php": "^7.2",
+        "ext-memcached": "*",
+        "psr/simple-cache": "~1.0.0"
     },
     "require-dev": {
-        "yiisoft/yii-core": "^3.0@dev",
-        "yiisoft/di": "^3.0@dev",
-        "yiisoft/log": "^3.0@dev",
-        "phpunit/phpunit": "^7.3",
-        "hiqdev/composer-config-plugin": "^1.0@dev"
+        "phpunit/phpunit": "^8.2"
     },
     "autoload": {
         "psr-4": {
-            "Yiisoft\\Cache\\": "src"
+            "Yiisoft\\Memcached\\": "src"
```

Comment:
I'd like to adjust namespaces before tagging releases: https://github.com/yiisoft/docs/blob/master/004-namespaces.md